cms/static/sass/elements/_modules.scss

```diff
@@ -168,18 +168,45 @@
 
     // specific menu types
     &.new-component-problem {
-      padding-bottom: ($baseline/2);
-
       .problem-type-tabs {
         display: inline-block;
       }
     }
+
+    .support-documentation {
```

Comment:
@alisan617 Could you do a quick review of my sass changes? Unfortunately I'm having to override some things that I would have inherited from our base sass because we use a JQuery widget that defines its own styling.